### ISSUE TITLE
Runner problem on environment with Alarm feature

### ISSUE THREAD

**Issue by marota** [rte-france/Grid2Op#216]
## Environment
 - Grid2op version: `'1.6.0.rc1'`
 - System: ` osx`
 - Additional system information

## Bug description
The runner is not starting properly when running on an envirnement with alarm feature activated

## How to reproduce

Use l2rpn_neurips_2020_track1_with_alert environment.

### Code snippet

```python
import grid2op
from grid2op.Agent import DoNothingAgent
from grid2op.Runner import Runner
from tqdm import tqdm

###change YourPath here
path_env='YourPath/l2rpn_neurips_2020_track1_with_alert'

env = grid2op.make(path_env)

runner = Runner(**env.get_params_for_runner(),
                agentClass=DoNothingAgent
                )
res = runner.run(nb_episode=1, 
                 nb_process=1,
                 pbar=tqdm)

... # Some code 
```

## Current output

```
Grid2OpException: Grid2OpException "Impossible to use the "AlarmReward" with an environment for which this feature is disabled. Please make sure "env._has_attention_budget" is set to ``True`` or change the reward class with `grid2op.make(..., reward_class=AnyOtherReward)`"

```

## Expected output

```
The runner should run over the scenario
```

**Comment by BDonnot** [comment]
By the way, Issue is fixed, new changes can be obtained by intalling the dev version of grid2op:

`pip install git+git://github.com/rte-france/grid2op.git@dev_1.6.0`

**Comment by BDonnot** [comment]
Fixed and merged in version 1.6.0, now available on pypi
